apprentice_teacher.rb

```diff
@@ -1,53 +1,23 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'Teacher'
+
+class ApprenticeTeacher < Teacher
+  include Teachable
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
-    @phase = 3
-  end
-
-  def offer_high_five
-    "High five!"
+    @benchmark_rating = 80
```

Comment:
What are the tradeoffs between doing this as an instance variable vs constant? Why choose one over the other?